internal/pkg/manifest/validate.go

```diff
@@ -1721,6 +1738,47 @@ func validateDepsForEssentialContainers(deps map[string]containerDependency) err
 	return nil
 }
 
+func validateExposedPorts(opts validateExposedPortsOpts) error {
+	exposedPorts := make(map[uint16]string)
+
+	if opts.mainContainerPort != nil {
+		exposedPorts[aws.Uint16Value(opts.mainContainerPort)] = opts.mainContainerName
+	}
+
+	for name, sidecar := range opts.sidecarConfig {
+		if sidecar.Port != nil {
+			port, err := strconv.Atoi(aws.StringValue(sidecar.Port))
```

Comment:
does `Atoi` work if the port is `8080/tcp`?

Reply:
Hmm good catch. I will parse it before directly using it.

Reply:
it make work fine, i just wasn't sure!

Reply:
ok jk i just tested and it doesn't work :)